src/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Provides an implementation of Application Insights telemetry processor that feeds Application Insights telemetry into EventFlow pipeline. 
+    This allows sending diagnostics data from applications instrumented with Application Insights to destinations other than Application Insights service.</Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <Authors>Microsoft</Authors>
+    <TargetFrameworks>net451</TargetFrameworks>
+    <DelaySign>true</DelaySign>
+    <AssemblyName>Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../PublicKey.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageId>Microsoft.Diagnostics.EventFlow.EventFlowTelemetryProcessor</PackageId>
```

Comment:
package name should indicate that it is used for Application Insights. `Microsoft.Diagnostics.EventFlow.ApplicationIsnightsSink` ?